### PR TITLE
feat(ui):Adjust the toast when changing anything in Settings

### LINKED ISSUE #2465: [UI] Adjust the toast when changing anything in Settings
### Issue description

When changing anything in Settings, the toast says "is successfully saved" without saying what.

We can simply change the message to "Settings have been saved".

image

### PR DESCRIPTION
**Pull request by EdwardLi-coder**
closes #2465

### CODE REVIEW

**Review by anna-geller — APPROVED** [review]
LGTM, thanks for fixing this!

File: ui/src/components/settings/BasicSettings.vue [hunk 1]
```diff
@@ -223,7 +223,7 @@
                 } else {
                     localStorage.removeItem("defaultNamespace")
                 }
-                this.$toast().saved();
```
Comment (brian-mulier-p) [inline ui/src/components/settings/BasicSettings.vue:226]: I'm not sure it's worth creating a new type of toast.
Maybe an alternative would be that you use the options from the saved method to give an "genericQualifier" key which defines whether or not we should display the saved thing in italic (false = italic, which will be the default not to break anything).

You can then update the saved() method to check for this param (don't forget to be null-safe on the options param :+1: ). Conditionally on that you can create a new translation that **doesn't includes** the `<em>` (see [here](https://github.com/kestra-io/kestra/blob/develop/ui/src/translations.json#L27)) if this param is true.

This way we now have a customizable toast which allow to display saved things whether they are id or not (and so the display will be adapted to it).

An example of usage for settings saving would then be
`this.$toast().saved(this.$t('setting'), undefined, {genericQualifier: true})`

To recap:

- in saved method check for options?.genericQualifier
- if true -> use $t('generic saved done') else -> use $t('saved done')
- create a translation for `generic saved done` key [here](https://github.com/kestra-io/kestra/blob/develop/ui/src/translations.json)
- use `this.$toast().saved(this.$t('setting'), undefined, {genericQualifier: true})` everywhere a setting is saved
- remove your introduced "toast().settingsSaved()"

**Review by brian-mulier-p — CHANGES_REQUESTED** [review]

File: ui/src/components/settings/BasicSettings.vue (hunk 1 above)
Comment (EdwardLi-coder) [reply ui/src/components/settings/BasicSettings.vue:226]: Ok.I will update it.

File: ui/src/translations.json [hunk 2]
```diff
@@ -25,6 +25,7 @@
     "save": "Save",
     "saved": "Successfully saved",
     "saved done": "<em>{name}</em> is successfully saved",
+    "generic saved done": "Settings have been saved",
```
Comment (brian-mulier-p) [inline ui/src/translations.json:28]:
```suggestion
    "generic saved done": "{name} have been saved",
```

File: ui/src/utils/toast.js [hunk 3]
```diff
@@ -41,16 +41,19 @@ export default {
                 },
                 saved: function(name, title, options) {
                     ElNotification.closeAll();
+                    const message = options && options?.genericQualifier
+                        ? self.$t("generic saved done")
```
Comment (brian-mulier-p) [inline ui/src/utils/toast.js:45]:
```suggestion
                        ? self.$t("generic saved done", {name})
```

File: ui/src/components/settings/BasicSettings.vue [hunk 4]
```diff
@@ -223,7 +223,7 @@
...
+                this.$toast().saved(undefined,undefined,{genericQualifier: true});
```
Comment (brian-mulier-p) [inline ui/src/components/settings/BasicSettings.vue:226]:
```suggestion
                this.$toast().saved(this.$t('settings'),undefined,{genericQualifier: true});
```

**Review by brian-mulier-p — CHANGES_REQUESTED** [review]
some little changes and that's it

File: ui/src/utils/toast.js [hunk 5]
```diff
@@ -41,16 +41,19 @@ export default {
                 },
                 saved: function(name, title, options) {
                     ElNotification.closeAll();
+                    const message = options && options?.genericQualifier
```
Comment (brian-mulier-p) [inline ui/src/utils/toast.js:44]:
```suggestion
                    const message = options?.multiple
```

**Review by brian-mulier-p — CHANGES_REQUESTED** [review]

File: ui/src/utils/toast.js (hunk 5 above)
Comment (EdwardLi-coder) [reply ui/src/utils/toast.js]: Thank you.I update it.

**Review by brian-mulier-p — APPROVED** [review]
LGTM, I'll merge it, thank's for your contribution :)
